Modules/LinearBoltzmannSolver/lbs_linear_boltzmann_solver.h

```diff
@@ -138,7 +138,7 @@ class Solver : public chi_physics::Solver
                  bool suppress_phi_old = false);
   double ComputePiecewiseChange(LBSGroupset *groupset);
   SweepChunk *SetSweepChunk(int group_set_num);
-  void ClassicRichardson(int group_set_num, SweepChunk* sweep_chunk, MainSweepScheduler & sweepScheduler, bool log_info = true);
+  bool ClassicRichardson(int group_set_num, SweepChunk* sweep_chunk, MainSweepScheduler & sweepScheduler, bool log_info = true);
```

Comment:
I completely understand the bool here. Let's be consistent and also change GMRES to return the converged logic.

Reply:
done